### DISCUSSION TITLE
Is there a way for excalidraw files to be opened by other software such as Visio?

### DISCUSSION THREAD

**Discussion by aravindk100** [excalidraw/excalidraw#5318] [General]
While I love using Excalidraw, others I work with are more in Visio world. Is there any interoperability that is feasible? I did notice a thread back in 2021 that said this was not supported. I am wondering if there is a road map or third party way to do this to avoid having to recreate in Visio.

**Comment by dwelle** [comment]
Hi. What would be the ideal workflow you have in mind for this?
